src/modules/ewon-flexy-registration/bulk-registration/bulk-registration.component.ts

```diff
@@ -122,6 +122,71 @@ export class BulkRegistrationComponent implements OnInit {
   ngOnDestroy(): void {
     // this.flexyRegistration.closeSubscriptionOnDeviceRequestRegistration();
   }
+
+  private async deleteExistingRequests() {
+    const existingRequests = await this.flexyRegistration.getDeviceRequestRegistration();
+    const promises = this.selectedItems
+      .filter(item => existingRequests.find(element => element.id == item.toString()))
+      .map(item => this.flexyRegistration.deleteDeviceRequestRegistration(item.toString()));
+    await Promise.all(promises);
+  }
```

Comment:
I guess deleting these requests does not make sense at all.. especially if the later on skipping those which are in the Not_integrated state..